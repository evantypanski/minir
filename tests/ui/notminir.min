this is not a minir file.
