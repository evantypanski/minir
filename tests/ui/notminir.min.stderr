error: at tests/ui/notminir.min:1: expected 'func' token
this is not a minir file.


Found 1 errors during parsing
error: Fatal error occurred; aborting
